pinot-tools/src/main/java/com/linkedin/pinot/tools/admin/command/AvroSchemaToPinotSchema.java

```diff
@@ -69,6 +69,11 @@
   public boolean execute() throws Exception {
     Schema schema = null;
 
+    if (_dimensions == null && _metrics == null) {
```

Comment:
These are not mutually exclusive, right? If so, we cannot use forbid. If args4j does not have a feature to have at least one of two (or more) as required, then go ahead and use the 'if' check you had originally.